dnsmasq/run.sh

```diff
@@ -4,6 +4,19 @@ set -e
 CONFIG="/etc/dnsmasq.conf"
 
 bashio::log.info "Configuring dnsmasq..."
+
+if ! bashio::config.is_empty 'interface';
```

Comment:
```suggestion
if bashio::config.has_value 'interface'; then
```

Using `has_value` removes the need for a negative expression. `then` can be placed on the same line.

Same change for all other occurrences

Reply:
fixed in recent commit